src/index.html

```diff
@@ -11,12 +11,105 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container header-container">
+        <a
+          href="#"
+          class="logo"
+        >
+          <img
+            src="images/logo.png"
+            alt="'Moyo Company Logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul>
+            <li class="nav_item">
+              <a
+                href="#"
+                class="nav_link is-active apple"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                href="#"
+                class="nav_link"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                href="#"
+                class="nav_link"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                href="#"
+                data-qa="hover"
+                class="nav_link lc"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                href="#"
+                class="nav_link"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                href="#"
+                class="nav_link"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                href="#"
+                class="nav_link"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav_item">
+              <a
+                href="#"
```

Comment:
The `href` attributes for the navigation links are set to `#`, which is a placeholder. Consider replacing them with actual URLs or handling navigation with JavaScript if it's for a single-page application.